source/scss/_pf-component-view-selector.scss

```diff
@@ -0,0 +1,15 @@
+//
+// View Selector component
+//
+
+.pf-view-selector__btn-link {
+  color: $color-pf-black-900;
+  font-size: $font-size-relative-base;
+  padding: 0;
```

Comment:
I'd rather solve that in a later story because I don't think we've solved this for buttons in general, have we? If there's a min-size on buttons, then it should flow through to here